cmd/queue/main.go

```diff
@@ -93,6 +93,7 @@ func initEnv() {
 	servingNamespace = util.GetRequiredEnvOrFatal("SERVING_NAMESPACE", logger)
 	servingRevision = util.GetRequiredEnvOrFatal("SERVING_REVISION", logger)
 	servingAutoscaler = util.GetRequiredEnvOrFatal("SERVING_AUTOSCALER", logger)
+	autoscalerNamespace = util.GetRequiredEnvOrFatal("AUTOSCALER_NAMESPACE", logger)
```

Comment:
Can we use the environment value to transmit the autoscaler namespace?@mattmoor

Reply:
I don't understand why this is different from `system.Namespace()`, I'd rather chase the root cause than work around it like this.